src/MuTalk-Tests/MutationTestingFixedNumberOfMutantsBudgetTest.class.st

```diff
@@ -0,0 +1,51 @@
+Class {
+	#name : 'MutationTestingFixedNumberOfMutantsBudgetTest',
+	#superclass : 'MutationTestingBudgetTest',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'running' }
+MutationTestingFixedNumberOfMutantsBudgetTest >> runAnalysisFor: aNumber [
```

Comment:
Hmm I wonder if we cannot do a better template method here avoiding repeating code between the sibling classes...